lib/search_solr_tools/harvesters/tdar.rb

```diff
@@ -14,14 +14,22 @@ def harvest_and_delete
       end
 
       def harvest_tdar_into_solr
-        start_record = 1
+        start_record = 0
+        total_harvested = 0
+        total_expected = total_results
```

Comment:
It is - https://github.com/bbatsov/ruby-style-guide#method-invocation-parens

And rubocop doesn't like it:

> ```
> ber rubocop
Running RuboCop...
Inspecting 107 files
.......................C...................................................................................

Offenses:

lib/search_solr_tools/harvesters/tdar.rb:19:39: C: Do not use parentheses for method calls with no arguments.
        total_expected = total_results()
                                      ^

107 files inspected, 1 offense detected
RuboCop failed!
```